include/mxnet/c_api.h

```diff
@@ -218,25 +218,35 @@ MXNET_DLL int MXNotifyShutdown();
  * \param num_params Number of parameters
  * \param keys array of parameter keys
  * \param vals array of parameter values
+ * \param kvstoreHandle handle to kvstore
  * \return 0 when success, -1 when failure happens.
  */
-MXNET_DLL int MXSetProfilerConfig(int num_params, const char* const* keys, const char* const* vals);
+MXNET_DLL int MXSetProfilerConfig(int num_params, const char* const* keys, const char* const* vals,
```

Comment:
C API, so shouldn’t pass default arguments

Reply:
if you do, it should be with some macro that will be nothing in C. because if you include this header from a C file, it will choke on that default argument

Reply:
Oh I see, will remove this

Reply:
Ok, will do. Then does it technically change the C API? Should I create wrapper functions which call the new function by passing null to this new function?